prime-router/src/main/kotlin/fhirengine/translation/hl7/schema/ConfigSchema.kt

```diff
@@ -126,12 +126,10 @@ abstract class ConfigSchema<T : ConfigSchemaElement>(
             // Why the distinct? A schema can make references to the same schema multiple times, so you could get
             // a list of elements that are identical, so we make sure to get only those that at different.
             elementsInSchema = elements.filter { it.schemaRef != null }.mapNotNull {
-                it.schemaRef?.findElement(elementName)
-            }.distinct()
+                it.schemaRef?.findElements(elementName)
+            }.flatten().distinct()
         }
-        // Sanity check
-        check(elementsInSchema.size <= 1)
```

Comment:
The need for this restriction was unclear to me and now that an observation-result can appear in two places was actually causing problems.  The California transform overrides `observation-note` so it is always false.  Now that `observation-result` is used in `order-observation` and `specimen` this code failed as `observation-note` now appeared twice in the schema.

The change was to return all the found elements and merge the override into all of them.  This made sense in the example above, where the assumption would be that California wants `observation-note` suppressed everywhere.

Reply:
What you're describing makes sense, but it concerns me a little bit when  I think about what happens when the element that appears twice isn't overridden. There are checks for duplicate elements in the convert and translate steps (search for usages of `duplicateElements`), would those fail if `observation-result` for example isn't overridden? Coming off of vacation, I'm having a hard time wrapping my mind around the details of schema extension vs schema reference within an element, so it might not be a real concern, and I can probably test it on my own too.

Reply:
Also, if it is a problem, we might be able to fix that by just not checking for "duplicate elements" in those steps (duplicates are really just elements with the same name). It almost seems like this was enforced in schema merging and in the other steps to meet an arbitrary assumption that names would be unique in a schema tree. You point out a good reason to allow named to be repeated, and I can't think of a great reason not to allow that.

Reply:
I think error wise this actually works as expected as only the California translation overrides `observation-result` but the base one does not.  I can take at those usages and try to see why that isn't causing an error.